README.md

```diff
@@ -169,6 +169,24 @@ takeout stop {container_id}
 takeout stop {container_id1} {container_id2}
 ```
 
+### Get a shell inside any Takeout container
+
+To get a shell inside any container that is started with Takeout, you may run:
+
+```bash
+takeout shell {service}
+```
+
+Here are some examples:
+
+```bash
+takeout shell mysql
+takeout shell neo4j
+takeotu shell pgvector
+```
+
+This will reuse the running container. Takeout will start either a `bash` or a `sh` process inside the container, depending on what the container supports (this is automatically handled by Takeout).
```

Comment:
```suggestion
This will open a shell inside the running container for the service you provide. Takeout will start either a `bash` or a `sh` process inside the container, depending on what the container supports.
```